test/component/testThankYouPage.js

```diff
@@ -15,7 +15,23 @@ describe('thank-you', () => {
     });
 
     describe('Verify Content, Errors and Redirection', () => {
-        it('test content loaded on the page', (done) => {
+        it('test content loaded on the page when CCD Case ID not present', (done) => {
+            const sessionData = {};
+            testWrapper.agent.post('/prepare-session/form')
+                .send(sessionData)
+                .end(() => {
+                    const excludeKeys = ['referenceNumber'];
+
+                    const contentData = {
+                        helpLineNumber: config.helpline.number,
+                        findOutNext: config.links.findOutNext
+                    };
+
+                    testWrapper.testContent(done, excludeKeys, contentData);
```

Comment:
It would be better to use `testWrapper.testContentNotPresent()` in this test because excluding the content will make the test pass by simply not checking that it exists in the page.

For example, if you excluded the same content in the `test content loaded on the page when CCD Case ID present` test, where this content should appear, then the test would still pass.